boards/arm/verdin_imx8mp_m7/verdin_imx8mp_m7_defconfig

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2023 Toradex
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_IMX8ML_M7=y
+CONFIG_SOC_MIMX8ML8=y
+CONFIG_BOARD_VERDIN_IMX8MP_M7=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_CONSOLE=y
+CONFIG_XIP=y
+CONFIG_PINCTRL=y
+
+CONFIG_CODE_ITCM=y
+CONFIG_CODE_DDR=n
```

Comment:
Is `CONFIG_CODE_DDR=n` needed here? This symbol is a Kconfig choice.

Reply:
I'm not sure if this is the right approach, but I added both configs `CONFIG_CODE_ITCM` (enabled by default) and `CONFIG_CODE_DDR` (disabled by default) as options, as explained in the `index.rst`. If the user chooses to use the ITCM RAM, leave it as is. If it decides to use DDR RAM, enable the `CONFIG_CODE_DDR=y` and CONFIG_CODE_ITCM=n`.

Reply:
Changed to two different targets, one for DDR and the other for TCM, as explained on https://github.com/zephyrproject-rtos/zephyr/pull/61713#discussion_r1376573982